aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -1006,6 +1006,70 @@ void getrfBatched<c10::complex<float>>(
       batchsize));
 }
 
+template <>
+void getrsBatched<float>(CUDABLAS_GETRS_ARGTYPES(float)) {
```

Comment:
ROCm CI fails to build because there is no getrsBatched function in rocBLAS. These functions should be moved to be under `#ifdef CUDART_VERSION`. See the code for geqrfBatched, getrfBatched, getriBatched.